tests/components/error_handling.test.ts

```diff
@@ -264,6 +264,30 @@ describe("errors and promises", () => {
     expect(error!.message).toBe("Tokenizer error: could not tokenize `{ 'invalid: 5 }`");
   });
 
+  test("wrapped errors in async code are correctly caught", async () => {
+    class App extends Component {
```

Comment:
in owl 2 code, I tried to name all `App` component `Root` instead, to avoid confusion with the new `App` class